doc/cephfs/shell.rst

```diff
@@ -0,0 +1,277 @@
+
+Ceph FS Shell
+=============
+
+The File System (FS) shell includes various shell-like commands that directly interact with the Ceph File System.
```

Comment:
See: https://github.com/ceph/ceph/pull/22784#pullrequestreview-135138797 and follow it everywhere else.

Reply:
Thanks! I will work on it.

Reply:
1. Put the doc changes in a separate commit.
2. The commit lack `Fixes:` tag. See [Tag the Commit](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#tag-the-commit)
3. Update the tracker with the PR url and change the status to Needs Review.